### PR TITLE
Fixes typos, whitespace, markdown

### PR DESCRIPTION
**Pull request by nikoandpiko**
Fixes some typos, removes whitespace, suggests markdown improvements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,35 +6,38 @@ Just like everyone can use mind mapping software to organize ideas and improve e
 
 XMindMark files are written in plain text, which means that you can describe a mind map faster using any text editor at any time anywhere, like:
 
-```
+```XMindMark
```
Comment (parrotdance) [inline README.md]: Thanks for your suggestion, but I think lower-case version `xmindmark` will be better, for all similar code blocks.

File: README.md [hunk 2]
```diff
@@ -89,9 +94,10 @@ In the case above, a `.svg` file will be generated in your working directory.
 > xmindmark --outputDir ./out <your-xmindmark-file>
 ```
 
-You can use `-o` or `--outputDir` flag to specify the output directory where the generated file will be placed. Both relative and absolute path are supported.
+You can use `-o` or `--outputDir` flag to specify the output directory where the generated file will be placed. Both relative and absolute paths are supported.
```
Comment (parrotdance) [inline README.md]: It shall be described here is ***"Either relative or absolute paths are supported."***

File: README.md [hunk 3]
```diff
@@ -135,7 +142,7 @@ Currently, xmindmark only supports converting `string` content to [`ArrayBuffer`
 Converting XMindMark files to `.svg` files relies on a graphical environment, so when xmindmark converts an XMindMark file to a `.svg` file for the every first time, it must complete the following preparations:
 
 1. Download a *bundled version* (specified [here](https://github.com/puppeteer/puppeteer/#q-which-chromium-version-does-puppeteer-use)) of Chromium by [puppeteer-core](https://www.npmjs.com/package/puppeteer-core).
-2. Download the mind map rendering engine from XMind official website: https://assets.xmind.net.
+2. Download the mind map rendering engine from XMind official website: <https://assets.xmind.net>.
```
Comment (parrotdance) [inline README.md]: There is no need for brackets here, use web site directly is okay.

**Comment by nikoandpiko** [comment]
@parrotdance Thank you for the feedback. The latest commit addresses the above points.

**Comment by parrotdance** [comment]
lgtm

**Pull request merged**